mule-migration-tool-library/src/main/resources/report.yaml

```diff
@@ -997,6 +997,11 @@ basicStructure:
     type: WARN
     message: "Configuration item in policy YAML does not have property '{propertyName}', it was set by default to value '{defaultValue}'."
 
+apiDiscovery:
+  apiId:
+    type: WARN
+    message: "APIs in Mule 4 have an identifier, the equivalent should be: '{apiName}:{apiVersion}'."
```

Comment:
Could we add the link to the documentation on Autodiscovery for Mule 4? https://docs.mulesoft.com/api-manager/2.x/configure-autodiscovery-4-task

Reply:
LGTM